localstack-core/localstack/openapi.yaml

```diff
@@ -481,6 +507,8 @@ paths:
   /_aws/sqs/messages/{region}/{account_id}/{queue_name}:
     get:
       description: List SQS messages without side effects
+      operationId: list_sqs_queue_messages
```

Comment:
any idea if there is support for polymorphism and overloading? this operation and the previous are the same

if not, the operation ID could be set as follows to reduce confusion

`list_all_sqs_messages`
`list_sqs_messages`